tendrl/node_agent/manager/manager.py

```diff
@@ -82,6 +84,18 @@ def _run(self):
         LOG.info("%s complete" % self.__class__.__name__)
 
 
+def update_node_context(manager, machine_id, tags=""):
```

Comment:
This patch has been verified with, the patch you mentioned and it does not have any implication on that.